tests/load/gcp/deploy.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
```

Comment:
Overall, this definitely wants to be using something like Terraform, but for now I'll drop some hints to make your bash script less likely to explode.

Reply:
I agree with @jnsgruk that we'll want to turn this into terraform at some point. Would you mind raising an issue for that, @sed-i, so we can revisit that later?